service/RoleBandService.ts

```diff
@@ -0,0 +1,14 @@
+import { RoleBandResponse } from "../model/RoleBandResponse";
+const axios = require('axios');
+
+axios.defaults.baseURL = process.env.API_URL;
+
+module.exports.getRoleBands = async function (): Promise<RoleBandResponse> {
+    try {
+        const response = await axios.get(`${this.baseURL}/api/role-band-levels`)
```

Comment:
If we define the base URL as a variable like above then I don't believe that we will need this this. 

```suggestion
        const response = await axios.get(`${baseURL}/api/role-band-levels`)
```